include/iceflow/contentType.hpp

```diff
@@ -0,0 +1,10 @@
+#ifndef CONTENT_TYPE_HPP
+#define CONTENT_TYPE_HPP
+#include <cstdint>
+// ContentType header for defining custom TLV types
+
```

Comment:
You could also use a proper Doxygen comment here:

```suggestion
/**
 * ContentType header for defining custom TLV types
 */
```